samples/src/menger_sponge.cpp

```diff
@@ -54,6 +54,12 @@ Manifold MengerSponge(int n) {
   result -= hole;
   hole = hole.Rotate(0, 0, 90);
   result -= hole;
+
+  // Alternative order causes degenerate triangles
+  // Manifold tmp1 = hole.Rotate(90) + hole.Rotate(90).Rotate(0, 0, 90);
+  // Manifold tmp2 = hole + tmp1;
+  // result = result - tmp2;
```

Comment:
I'm leaving this here because it was the order that reproed the problem in #171 where all this was first noticed. It actually does still have 38k degenerate triangles, but the rest is correct now, so I'll look into that next.